client/lib/analytics/constants.js

```diff
@@ -0,0 +1,12 @@
+/** @format */
+
+export const NO_BLOG_ID_PATHS = [
+	// '/', Root path case handled manually
+	'/me',
```

Comment:
There are some site specific me paths 😄For example the billing/purchases section

Reply:
Could we maybe scan instead for if the following is in the path `/:site`? We probably don't want to pass any trailing selectedSite for an all site view.

Reply:
For example `/comments` shouldn't have a blog_id passed, but `/comments/all/:site` should

Reply:
> There are some site specific me paths 😄For example the billing/purchases section

So, thinking about it, the purchases site fragment does not make the path site-specific.
The site fragment is a qualifier of the purchase, but the owner of the purchase is the user.

Reply:
Well it still sets it in redux! 😆Purchases are a little funny they are per user and per site.

<img width="587" alt="screen shot 2018-03-28 at 4 49 29 pm" src="https://user-images.githubusercontent.com/1270189/38062291-081a4cea-32a8-11e8-918c-bc01002e8559.png">

Reply:
Even if that wasn't the case with purchases, we'd still may be reporting incorrectly for all site paths.

Reply:
> Well it still sets it in redux

It does set it, but does it actually use it? 😄
Either way, I guess I could do an exception for `/me/purchases/` with a `:site` fragment? 🤔

> we'd still may be reporting incorrectly for all site paths

All Sites already don't report `blog_id`, because `state.ui.selectedSiteId` gets `null`-ified.

Reply:
I'm honestly not sure why there's that much going on in the purchases url :). Theoretically if we have the purchase ID I believe we can lookup the site too.

> All Sites already doesn't report blog_id, because state.ui.selectedSiteId gets null-ified.

🤔 Let me test that out

Reply:
Looks good 👍